package.json

```
@@ -3,16 +3,17 @@
   "version": "1.0.7",
   "description": "Drag and drop task list items.",
   "repository": "github/task-lists-element",
-  "main": "dist/index.umd.js",
-  "module": "dist/index.esm.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "type": "module",
+  "types": "dist/index.d.ts",
   "scripts": {
     "clean": "rm -rf dist",
     "lint": "github-lint",
-    "prebuild": "npm run clean && npm run lint",
-    "build": "rollup -c && cp src/task-lists-element.js.flow dist/index.esm.js.flow && cp src/task-lists-element.js.flow dist/index.umd.js.flow",
+    "prebuild": "npm run clean && npm run lint && tsc",
```

Should this have `--noEmit`?

```suggestion
    "prebuild": "npm run clean && npm run lint && tsc --noEmit",
```

---

The is generating ds and compiling js to ts for rollup.